Apps/Src/main.c

```diff
@@ -11,9 +11,10 @@
 #include "MotorController.h"
 #include "Pedals.h"
 #include "CAN_Queue.h"
-
 #include "UpdateDisplay.h"
 
+#define IGN_CONT_PERIOD 10
```

Comment:
Any particular reason why we do this every 10 milliseconds? That seems excessive.

Reply:
artefact of old code, my b. I'll change it to 100ms